node/src/validator/mod.rs

```diff
@@ -82,6 +82,7 @@ impl<N: Network, C: ConsensusStorage<N>> Validator<N, C> {
         trusted_validators: &[SocketAddr],
         genesis: Block<N>,
         cdn: Option<String>,
+        dev_traffic: bool,
         storage_mode: StorageMode,
```

Comment:
nit: can you swap the order between `storage_mode` and `dev_traffic`?

Reply:
71230f240 Sure, though note `cargo fmt` then makes the comment indentation different:
<img width="1422" alt="Screenshot 2024-03-03 at 19 37 36" src="https://github.com/AleoHQ/snarkOS/assets/24724627/f36fe1a8-6996-4ebb-b428-8db6b5f33805">